node.go

```diff
@@ -71,30 +121,55 @@ func (nc *nodeCollector) Collect(ch chan<- prometheus.Metric) {
 }
 
 func (nc *nodeCollector) collectNode(ch chan<- prometheus.Metric, n api.Node) {
-	// Collect node conditions and while default to false.
-	// TODO(fabxc): add remaining conditions: NodeOutOfDisk, NodeMemoryPressure,  NodeDiskPressure, NodeNetworkUnavailable
-	for _, c := range n.Status.Conditions {
-		switch c.Type {
-		case api.NodeReady:
-			nodeStatusMetrics(ch, descNodeStatusReady, n.Name, c.Status)
-		}
+	addGauge := func(desc *prometheus.Desc, v float64, lv ...string) {
+		lv = append([]string{n.Name}, lv...)
+		ch <- prometheus.MustNewConstMetric(desc, prometheus.GaugeValue, v, lv...)
 	}
-
 	// NOTE: the instrumentation API requires providing label values in order of declaration
 	// in the metric descriptor. Be careful when making modifications.
-	ch <- prometheus.MustNewConstMetric(
-		descNodeInfo, prometheus.GaugeValue, 1,
-		n.Name,
+	addGauge(descNodeInfo, 1,
 		n.Status.NodeInfo.KernelVersion,
 		n.Status.NodeInfo.OSImage,
 		n.Status.NodeInfo.ContainerRuntimeVersion,
 		n.Status.NodeInfo.KubeletVersion,
 		n.Status.NodeInfo.KubeProxyVersion,
 	)
+
+	// Collect node conditions and while default to false.
+	// TODO(fabxc): add remaining conditions: NodeMemoryPressure,  NodeDiskPressure, NodeNetworkUnavailable
```

Comment:
Was wondering the same – that's probably set if heartbeats to the node fail at a connection level.

What's holding us back is that the vendored deps do not yet contain the constants for these conditions and godep refuses to update the vendoring on my machine without any indication of reason :)